README.md

```diff
@@ -2466,7 +2466,7 @@ Subscribe to an event published by the `tokenManager`.
 * `event` - Event to subscribe to. Possible events are:
   * `expired` - Fired before a token is set to expire (using `expireEarlySeconds` option, 30 seconds by default). If `autoRenew` option is set to true, a listener will be attached to this event and an attempt will be made to renew the token when the event fires.
   * `error` - Fired when a token renew attempt has failed. This is a permanent error, and the token will be removed from storage.
-  * `added` - Fired when a new token has been added or a new token has been added that did not change the original.
+* `added` - Fired when a new token has been added or a token has been updated that did not change from the original value.
```

Comment:
Does this mean it fires when a token is added or updated, **whether or not** the update has changed the token value?

Reply:
im going off your unit tests 
https://github.com/okta/okta-auth-js/blob/dd664ebea67692b1878ee05ffc2c2bc2699e00f0/test%2Fspec%2FtokenManager.js#L1742

https://github.com/okta/okta-auth-js/blob/dd664ebea67692b1878ee05ffc2c2bc2699e00f0/test%2Fspec%2FtokenManager.js#L1759
looks like thats kinda what you're saying yeah?

Any suggestions on rewording?
open to a 
```suggestion suggestion...

That sound about right tho?